dashboard-ui/src/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "proxyURL": "http://localhost:3000",
+  "apikey": "e67b309d4587e57e93070ba49e5d33aaa09a9e6da4af9654f0708df6c738aabd"
```

Comment:
Are these intended to be development defaults?

Reply:
they'll also need to be used in production.